ekncontent/overrides/EosKnowledgeContent.js

```diff
@@ -89,7 +89,9 @@ function _init() {
     });
     define_property(Eknc.ArticleObjectModel, 'table-of-contents', {
         get: function () {
-            let toc = this.get_table_of_contents().deep_unpack();
+            let toc = this.get_table_of_contents();
+            if (toc)
+                toc = toc.deep_unpack();
```

Comment:
I thought those were different checks, I mean, toc could be null but it could also be something "empty" (but we only know that after we unpack it?) , so If I get rid of the first  "if" I would only be checking the first null case, but not for the second case.